go.mod

```diff
@@ -34,38 +38,48 @@ require (
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/pborman/uuid v1.2.0 // indirect
 	github.com/pkg/errors v0.8.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0
 	github.com/prometheus/client_golang v0.9.2
 	github.com/sirupsen/logrus v1.3.0 // indirect
 	github.com/soheilhy/cmux v0.1.4 // indirect
 	github.com/spf13/afero v1.2.0 // indirect
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.3
+	github.com/stretchr/testify v1.4.0
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
 	github.com/ugorji/go/codec v0.0.0-20181209151446-772ced7fd4c2 // indirect
 	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
 	go.uber.org/atomic v1.3.2 // indirect
 	go.uber.org/multierr v1.1.0 // indirect
 	go.uber.org/zap v1.9.1 // indirect
-	golang.org/x/crypto v0.0.0-20190123085648-057139ce5d2b // indirect
+	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
 	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
-	google.golang.org/api v0.1.0
+	google.golang.org/api v0.6.1-0.20190607001116-5213b8090861
+	google.golang.org/grpc v1.23.1
 	gopkg.in/airbrake/gobrake.v2 v2.0.9 // indirect
 	gopkg.in/gcfg.v1 v1.2.3
 	gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2 // indirect
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.2.2 // indirect
 	gopkg.in/warnings.v0 v0.1.2
-	k8s.io/api v0.0.0-20190425012535-181e1f9c52c1
+	k8s.io/api v1.17.4
 	k8s.io/apiextensions-apiserver v0.0.0-20190312122106-cdf8f99e8934 // indirect
-	k8s.io/apimachinery v0.0.0-20190425132440-17f84483f500
-	k8s.io/apiserver v0.0.0-20190312120644-8bbd9f58a986
-	k8s.io/client-go v0.0.0-20190425172711-65184652c889
-	k8s.io/cloud-provider v0.0.0-20190313124351-c76aa0a348b5
-	k8s.io/component-base v0.0.0-20190311160731-923e373b2c7e
+	k8s.io/apimachinery v1.17.4
+	k8s.io/apiserver v1.17.4
+	k8s.io/client-go v1.17.4
+	k8s.io/cloud-provider v1.17.4
+	k8s.io/component-base v1.17.4
 	k8s.io/klog v0.3.0
-	k8s.io/kube-controller-manager v0.0.0-20190313124056-d2b16a081e75
-	k8s.io/kubernetes v1.14.1
-	sigs.k8s.io/structured-merge-diff v0.0.0-20190302045857-e85c7b244fd2 // indirect
+	k8s.io/kube-controller-manager v1.17.4
```

Comment:
these should be v0.17.4 to match k8s v1.17.4
ref: https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/2019-03-19-go-modules.md#select-a-versioning-strategy-for-published-modules